README.md

```diff
@@ -314,6 +314,8 @@ Table of Contents
   * [Shieldfy](https://shieldfy.io) — Web application firewall and vulnerability detection for developers, free plan up to 100k requests per month.
   * [Sqreen](https://www.sqreen.com/) — Application security monitoring and protection (RASP, WAF and more) for web applications and APIs. Free for 1 app and 3 million requests.
   * [Sucuri SiteCheck](https://sitecheck.sucuri.net) - Free website security check and malware scanner
+  * [Is it Hacked](http://isithacked.com) - Free cloud website security check and malware scanner. Email notifications for registered users.
```

Comment:
this one seems super minimal and barely maintained, I dont think it crosses the bar for the kind of quality site I'd look at

Reply:
Thanks for your comment, this product was designed to be minimal, but the security checks work well and we have have over 6900 registered users and over 20 000 websites in our database. So this is not a scam, this app really works well. We will add and SSL certificate to the website and we will create some more text content. This service is %100 free, which is not the case for most of the websites listed here. So please reconsider and accept it. Thanks.

Reply:
I appreciate the tests might be good, however this just isnt current or maintained from outside appearances, blog posts from the 2013 etc, links to hacking news with articles from 2016 etc.

Please remove it.

Reply:
I listed to you and I removed isithacked.com from the list. Thanks for your observations.